app/assets/javascripts/user.js

```diff
@@ -178,6 +220,42 @@ $(document).ready(function () {
     }
   }
 
+  function updateHomeLocation() {
+    const lat = $("#home_lat").val().trim();
+    const lon = $("#home_lon").val().trim();
+    if (!lat || !lon) {
+      return;
+    }
+
+    const geocodeUrl = `${OSM.NOMINATIM_URL}reverse?format=json&lat=${lat}&lon=${lon}`;
```

Comment:
If you only want a country why don't you ask for a country
```suggestion
    const geocodeUrl = `${OSM.NOMINATIM_URL}reverse?format=json&lat=${lat}&lon=${lon}&zoom=3`;
```
instead of getting a full address of things you don't care about and then doing `.split(",")`?

Reply:
Thank you for the suggested code. Was updated accordingly.